R/tbl_split.R

```diff
@@ -42,29 +42,30 @@
 #' trial |>
 #'   tbl_summary(by = trt) |>
 #'   tbl_split_by_rows(variables = c(marker, grade)) |>
-#'   tail(n = 1) # Print only last table for simplicity
+#'   dplyr::last() # Print only last table for simplicity
 #'
 #' # Example 2 ----------------------------------
 #' # Split by rows with row numbers
 #' trial |>
 #'   tbl_summary(by = trt) |>
 #'   tbl_split_by_rows(row_numbers = c(5, 7)) |>
-#'   tail(n = 1) # Print only last table for simplicity
+#'   dplyr::last() # Print only last table for simplicity
 #'
 #' # Example 3 ----------------------------------
 #' # Split by columns
 #' trial |>
 #'   tbl_summary(by = trt, include = c(death, ttdeath)) |>
 #'   tbl_split_by_columns(groups = list("stat_1", "stat_2")) |>
-#'   tail(n = 1) # Print only last table for simplicity
+#'   tail(n = 1) |> # Print only last table for simplicity
```

Comment:
I know you haven't requested review, so apologies if the comment is too early!

Is there a reason the other examples use `dplyr::last()` and this one uses tail and unlist?

Reply:
Sorry! this is ready from day one. I forgot the unlist here. Removing it